src/mood.ts

```diff
@@ -63,7 +64,17 @@ export function meatMood(urKels = false): Mood {
   mood.skill($skill`Drescher's Annoying Noise`);
   mood.skill($skill`Pride of the Puffin`);
 
-  if (myClass() !== $class`Pastamancer`) mood.skill($skill`Bind Lasagmbie`);
+  const mmjCost =
+    (100 -
+      (have($skill`Five Finger Discount`) ? 5 : 0) -
+      (have($item`Travoltan trousers`) ? 5 : 0)) *
+    (200 / (1.5 * myLevel() + 5));
+  const genericManaPotionCost = garboValue($item`generic mana potion`) * (200 / (2.5 * myLevel()));
```

Comment:
We want to use garboValue for things that we sell, but mallPrice for things that we buy

Reply:
Love the restructure